cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -321,13 +321,6 @@ describe('Data Table', () => {
   });
 
   describe('Truncation', () => {
```

Comment:
Can we perhaps add a test to make sure these new tooltip changes work as expected?

Reply:
Good catch, done!